server/config/settings/development.yml

```diff
@@ -0,0 +1,2 @@
+hosts:
+  - "http://localhost:8114"
```

Comment:
If localhost is default to all environments, it's better to set in `config/settings.yml`

Reply:
Thank you, done.